### PR TITLE
No u

### PR DESCRIPTION
**Pull request by k-k-karma**

### CODE REVIEW

File: config/admins.txt [hunk 1]
```diff
@@ -4,134 +4,4 @@
 #Ranks will match to those with the same name in admin_ranks.txt, if a match isn't found the user won't be adminned.
 #If SQL-based admin loading is enabled, admins listed here will always be loaded first and will override any duplicate entries in the database.
 
-Optimumtact = Host
```
Comment (TDcoolguy300) [inline config/admins.txt]: All of this.

**Comment by Jujumatic** [comment]
that's a lot of tags

**Comment by Jujumatic** [comment]
JESUS

**Comment by SireTurret** [comment]
what in the actual fuck

**Comment by Jujumatic** [comment]
and so the mighty beasts of the land waged war against each other... and there was much chaos and destruction..

**Comment by Jujumatic** [comment]
stop it!!!1

**Pull request closed without merging**